docs/product-assembly_pam-on-openshift/src/main/asciidoc/environment-managed-modify-proc.adoc

```diff
@@ -0,0 +1,18 @@
+[id='environment-managed-modify-proc']
+= Modifying a template for a managed environment
+
+To adjust the managed environment to your needs, you need to modify the template before deploying the environment.
+
+.Procedure
+. Select one of the templates to modify:
++ 
+* `rhpam70-sit.yaml` if you need to use minimal resources and do not need high-availability configuration for {CENTRAL} Monitoring and Smart Router.
```

Comment:
Trivial: Technically these two bulletted items should end with no period, since incomplete sentences. Per IBM SG.